src/carcockpit/shaders/shader_adv.hpp

```diff
@@ -22,25 +22,29 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #pragma once
 
 // #include <ruis/render/texturing_shader.hpp>
+#include <ruis/config.hpp>
 #include <ruis/render/opengles/shader_base.hpp>
 #include <ruis/render/texture_2d.hpp>
+#include <ruis/render/texture_cube.hpp>
 
-namespace ruis {
-using mat3 = r4::matrix3<real>;
-using matrix3 = mat3;
-static_assert(sizeof(mat3) == sizeof(real) * 3 * 3, "size mismatch");
-} // namespace ruis
-
-namespace carcockpit {
+namespace ruis::render {
 
 class shader_adv : public ruis::render::opengles::shader_base
```

Comment:
- rename `shader_adv` -> `shader_pbr`?
- add doxygen doc describing shortly what is PBR

Reply:
done